.github/workflows/ui-run-tests.yml

```diff
@@ -36,7 +36,6 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-          cache: 'npm'
```

Comment:
What issue are you running into? Using the cache will dramatically speed up PR checks so I'm hesitant to disable.

Reply:
Don't have the lockfile set up in the job yet; I want to quickly drop the cache so the job runs, get it passing, and then add the cache after the fact if possible. Happy to approach it from the other side.

Failing job example: https://github.com/GameMakersToolkit/team-finder/runs/3839966779?check_suite_focus=true

Reply:
Wait, I'm an idiot, turns out `cache-dependency-path` was what I was missing

Reply:
@dallonf have added the cache in correctly, but the runs that were no-cache (run #2) and cache (run #4) took about the same time. Looks like there's a bit more work to do, bear with

Reply:
Appears after a successful run
![image](https://user-images.githubusercontent.com/13312237/136587823-00836503-2560-446c-9686-b69eaba0f7be.png)